atd-vzv/src/views/map/Map.js

```diff
@@ -27,20 +27,9 @@ import "mapbox-gl/dist/mapbox-gl.css";
 import "@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw.css"; // Get out-of-the-box icons
 import MapInfoBox from "./InfoBox/MapInfoBox";
 import MapPolygonInfoBox from "./InfoBox/MapPolygonInfoBox";
+import MapGeocoder from "./Geocoder/Geocoder";
 
-const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNrM29wNnB3dDAwcXEzY29zMTU5bWkzOWgifQ.KKvoz6s4NKNHkFVSnGZonw`;
-
-function useMapEventHandler(eventName, callback, mapRef) {
-  useEffect(() => {
-    const currentMapRef = mapRef.current;
-    const mapDataListener = currentMapRef.on(eventName, function () {
-      callback();
-    });
-    return () => {
-      currentMapRef.off(eventName, mapDataListener);
-    };
-  }, [eventName, callback, mapRef]);
-}
+export const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNrM29wNnB3dDAwcXEzY29zMTU5bWkzOWgifQ.KKvoz6s4NKNHkFVSnGZonw`;
```

Comment:
Thanks for dropping that link, Frank! I agree that an issue to review and secure these tokens is in order since we fully control them. It would be good to push them out of the code and rotate them.

Reply:
Good looking out. I'll make an Issue for this.